src/fswAlgorithms/orbitControl/lambertSecondDV/_UnitTest/test_lambertSecondDV.py

```diff
@@ -0,0 +1,162 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+#
+import itertools
+
+import numpy as np
+import pytest
+from Basilisk.architecture import messaging
+from Basilisk.fswAlgorithms import lambertSecondDV
+from Basilisk.utilities import SimulationBaseClass
+from Basilisk.utilities import macros
+from Basilisk.utilities import orbitalMotion
+
+# parameters
+DVs = np.array([[1., 3., 4.], [500., -100., 200.]])
+time_maneuver = [1.e3, 2.e3]
+validFlag = [1]
+
+paramArray = [DVs, time_maneuver, validFlag]
+# create list with all combinations of parameters
+paramList = list(itertools.product(*paramArray))
+# add a case where lambert valid flag is false (0) to make sure module returns zeroed DV
+paramList.append((DVs[0], 1.e3, 0))
+
+@pytest.mark.parametrize("accuracy", [1e-4])
+@pytest.mark.parametrize("p1_dv, p2_tm, p3_valid", paramList)
+
+def test_lambertSecondDV(show_plots, p1_dv, p2_tm, p3_valid, accuracy):
+    r"""
+    **Validation Test Description**
+
+    This test checks if the Lambert Second DV module works correctly for different Delta-V vectors and maneuver times.
+    If the Lambert solution valid flag is false (0), the commanded Delta-V should be zero.
+
+    **Test Parameters**
+
+    Args:
+        :param show_plots: flag if plots should be shown
```

Comment:
remove `:param` to make the arguments render properly in HTML documentation

Reply:
Done.